packages/fxa-settings/src/components/Settings/PageRecoveryKeyCreate/index.tsx

```diff
@@ -2,49 +2,112 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { RouteComponentProps, useNavigate } from '@reach/router';
-import { useFtlMsgResolver } from '../../../models';
-import { usePageViewEvent } from '../../../lib/metrics';
-import VerifiedSessionGuard from '../VerifiedSessionGuard';
 import { HomePath } from '../../../constants';
+import { usePageViewEvent } from '../../../lib/metrics';
+import { useAccount, useFtlMsgResolver } from '../../../models';
+import FlowRecoveryKeyConfirmPwd from '../FlowRecoveryKeyConfirmPwd';
+import FlowRecoveryKeyDownload from '../FlowRecoveryKeyDownload';
 import FlowRecoveryKeyInfo from '../FlowRecoveryKeyInfo';
+import VerifiedSessionGuard from '../VerifiedSessionGuard';
+
+const viewName = 'settings.account-recovery';
 
 export const PageRecoveryKeyCreate = (props: RouteComponentProps) => {
-  const navigate = useNavigate();
+  usePageViewEvent(viewName);
+
+  const account = useAccount();
   const ftlMsgResolver = useFtlMsgResolver();
+  const navigate = useNavigate();
+
+  const [currentStep, setCurrentStep] = useState<number>(1);
+  const [formattedRecoveryKey, setFormattedRecoveryKey] = useState<string>('');
+
   const goHome = () => navigate(HomePath + '#recovery-key', { replace: true });
-  const [currentStep, setCurrentStep] = useState(1);
-  usePageViewEvent('settings.account-recovery');
+  const numberOfSteps = 4;
```

Comment:
You could move this to just after where you've got `viewName` set instead of scoping it to the page since it never changes and isn't dependent on a hook but 🤷‍♀️